tedana/selection/component_selector.py

```diff
@@ -65,20 +73,17 @@ def load_config(tree: str) -> Dict:
         )
         tree = "tedana_orig"
         fname = op.join(get_resource_path(), "decision_trees", tree + ".json")
-    else:
+    elif op.isfile(tree):
         fname = tree
+    else:
+        fname = download_json(tree, out_dir)
 
     try:
```

Comment:
```suggestion
    if fname and op.isfile(fname):
```
This can just be an if/else rather than a try/except. This means, if the file exists, but isn't a json, we'll get the error in `load_json` and if it doesn't exist we'll get the error here.